src/test/store/profile-view.test.js

```diff
@@ -1237,7 +1237,7 @@ describe('actions/ProfileView', function () {
             entryType: 'mark',
           },
         ],
-        ['c', 2, null],
+        ['c', 1023, null],
```

Comment:
This change so that a, b, c and d are sorted in this order (because they're sorted by start time).